src/src/CustomBuild/define_plugin_sets.h

```diff
@@ -3621,11 +3621,11 @@ To create/register a plugin, you have to :
 
 
   #ifndef FEATURE_THINGSPEAK_EVENT
-    #ifdef LIMIT_BUILD_SIZE
       #define FEATURE_THINGSPEAK_EVENT 0
-    #else
-      #define FEATURE_THINGSPEAK_EVENT 1
-    #endif
+  #endif
+
+  #ifndef FEATURE_OPENMETEO_EVENT
+    #define FEATURE_OPENMETEO_EVENT 0
   #endif
```

Comment:
Would be nice to have both feature flags enabled in the MAX builds 😃

Reply:
True! I just thought of it when I realized I was posting nonsense on the forum 😁.
Even though not many people use the max builds I guess...

Reply:
> not many people use the max builds I guess...

We _are_ trying to push more users to MAX builds... 😎

Reply:
Oh, and maybe they are also useful in the ESP32 Climate builds?